src/components/Proposal/ProposalHistoryRow.tsx

```diff
@@ -218,18 +219,22 @@ export default withSubscription({
     const proposal = props.proposal;
     if (!props.currentAccountAddress) {
       return combineLatest(
-        proposal.state(),
+        proposal.state({}),
         of([]),
         of([]),
         of(null),
       );
     } else {
+      const member = new Member(getArc(), Member.calculateId({
+        contract: props.daoState.reputation.id,
+        address: props.currentAccountAddress,
+      }));
+
       return combineLatest(
-        proposal.state(),
+        proposal.state({}),
```

Comment:
really?  `state` can't have a default param of `{}`?  I suppose maybe that helps people avoid doing it unintentionally.

Reply:
Agreed, made this issue here: https://github.com/daostack/arc.js/issues/479